xml-files/crr.xml

```diff
@@ -4,6 +4,7 @@
     <transaction duration="$duration">
       <flowop type="connect" options="remotehost=$remotehost protocol=$protocol port=$port"/>
       <flowop type="write" options="size=$wsize"/>
+      <flowop type="think" options="duration=$think"/>
```

Comment:
It seems "think" flowop has another option following the "duration" option. It is either "busy" or "idle". Should we also allow/require this option param.
May be w/o either of those options, it does not  wait. 

    } else if (strcasecmp(option, "busy") == 0) {
        flowop->options.flag |= O_THINK_BUSY;
        return (UPERF_SUCCESS);
    } else if (strcasecmp(option, "idle") == 0) {
        flowop->options.flag |= O_THINK_IDLE;
        return (UPERF_SUCCESS);

Above, no "busy" or no "idle",  no flag set.

Below, no flag set, no wait.

```
flowop_think(strand_t *sp, flowop_t *fp)
{
    int error;
    flowop_options_t *fo = &fp->options;
    uperf_info("thinking for %.2fms\n", 1.0*fp->options.duration/1.0e+6);
    if (FO_THINK_BUSY(fo)) {
        error = uperf_spin(fp->options.duration);
    } else {
        error = uperf_sleep(fp->options.duration);
    }

    return (error);
}
```

Reply:
I might mis-read. It should have a flowop_think, and the code this flowop default "else" to sleep.

Reply:
I am fine with idle behavior, but I don't have a problem supporting both.

Reply:
crap, overlooked that